synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -104,6 +115,41 @@ export class SynopsysToolsParameter {
       polData.data.polaris.triage = inputs.POLARIS_TRIAGE;
     }
 
+    if (
+      inputs.POLARIS_PROJECT_DIRECTORY ||
+      inputs.PROJECT_SOURCE_ARCHIVE ||
+      inputs.PROJECT_SOURCE_EXCLUDES ||
+      inputs.PROJECT_SOURCE_PRESERVE_SYM_LINKS
```

Comment:
none of these fields depend on each other?

Reply:
There is some dependency, as validation will be performed on the bridge side. So  right now we do not have to do anything about that.

Reply:
"If assessment mode is enabled then all these fields are applicable". Yes, you're correct. But as of our last discussion with Tithi, she said that we won't have to perform validation on our side. Even If the assessment mode is not enabled but the user provides all other fields, we will pass them to the bridge accordingly. The bridge will handle the rest of the things.

Reply:
even in case of pr comment or fix pr, we pass other params only the main input is enabled.